designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/activities/ScenarioActivityRepository.scala

```diff
@@ -86,6 +92,14 @@ object ScenarioActivityRepository {
     case object CouldNotModifyComment extends ModifyCommentError
   }
 
+  sealed trait DeleteAttachmentError
+
+  object DeleteAttachmentError {
+    case object ActivityDoesNotExist     extends DeleteAttachmentError
+    case object AttachmentDoesNotExist   extends DeleteAttachmentError
```

Comment:
not used?

Reply:
yes, removed, only CouldNotDeleteAttachment is used